gpii/node_modules/orca/orcaSettingsHandler.js

```diff
@@ -84,7 +84,7 @@
         var userSettings = require(orcaSettingsFile);
         var currentSettingsInProfile = userSettings.profiles[userSettings.general.activeProfile[PROFILE_ID]];
 
-        var applier = fluid.makeChangeApplier(currentSettingsInProfile, {
+        var applier = fluid.makeHolderChangeApplier(currentSettingsInProfile, {
```

Comment:
I am quite sure this will fail. If it passes, this implies a serious fault in the test cases which should be fixed - I imagine that the fault is that you don't attempt to test any removal of settings - although I can't imagine that the tests which try to verify that the user's original settings have been restored will pass - did you try them?

https://github.com/GPII/universal/blob/master/tests/platform/linux/linux-orca-testSpec.js#L25

Either way we have a serious problem that our test cases can't be testing what we hope they are testing